internal/services/mssql/helper/sql_virtual_machine_storage_settings.go

```diff
@@ -28,3 +28,52 @@ func StorageSettingSchema() *pluginsdk.Schema {
 		},
 	}
 }
+
+func SQLTempDBStorageSettingSchema() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"data_file_count": {
+					Type:     pluginsdk.TypeInt,
+					Optional: true,
+					Default:  8,
+				},
+				"data_file_size": {
```

Comment:
what units is this in?
```suggestion
				"data_file_size_mb": {
```